parm/config/config.base.emc.dyn

```diff
@@ -192,7 +189,19 @@ case "${APP}" in
     export CCPP_SUITE="FV3_GFS_v17_coupled_p8"
     export WAVE_CDUMP="both"
     export cplwav2atm=".true."
-    export confignamevarfornems="cpld${aero_nems_string}_wave"
+    export confignamevarfornems="cpld_wave"
+    source $EXPDIR/config.defaults.s2sw
+    ;;
+  S2SWA)
+    export DO_COUPLED="YES"
+    export DO_WAVE="YES"
+    export DO_OCN="YES"
+    export DO_ICE="YES"
+    export DO_AERO="YES"
+    export CCPP_SUITE="FV3_GFS_v17_coupled_p8"
+    export WAVE_CDUMP="both"
+    export cplwav2atm=".true."
+    export confignamevarfornems="cpld_aero_wave"
```

Comment:
This is not complete, but something along this lines of compressing to avoid duplication of big blocks.

```suggestion
  S2S |  S2SW | S2SWA)
    export DO_COUPLED="YES"
    if [[ $APP == "S2SW" || ${APP} == "S2SWA" ]]; then
      export DO_WAVE="YES"
      export WAVE_CDUMP="both"
      export cplwav2atm=".true."
      export confignamevarfornems="cpld_wave"
    fi
    if [[ $APP == "S2SWA" ]]; then
      export DO_AERO="YES"
      export confignamevarfornems="cpld_aero_wave"
    fi

```

Reply:
Updated